src/smc-webapp/editor.coffee

```diff
@@ -3126,6 +3157,10 @@ class JupyterNBViewerEmbedded extends FileEditor
         @iframe.maxheight()
 
 {HTML_MD_Editor} = require('./editor-html-md/editor-html-md')
+html_md_exts = []
+for ext, opts of file_associations
```

Comment:
Oh, this is the same as the (more readable) one-liner list comprehension:
```
html_md_exts = (ext for ext, opts of file_assocations when opts.editor == 'html-md')
```
This is like Python, but with paren instead of square brackets.  

That said, maybe exactly this snippet of code (suitably optimized) should actually be done automatically when registering editors.... something to worry about later.